service/src/test/java/greencity/mapping/EmployeeToOptionForColumnDtoMapperTest.java

```diff
@@ -0,0 +1,36 @@
+package greencity.mapping;
+
+import greencity.ModelUtils;
+import greencity.dto.OptionForColumnDTO;
+import greencity.entity.user.employee.Employee;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.transaction.annotation.Transactional;
+
+@ExtendWith(MockitoExtension.class)
+class EmployeeToOptionForColumnDtoMapperTest {
+
+    @InjectMocks
+    EmployeeToOptionForColumnDtoMapper employeeToOptionForColumnDtoMapper;
+
+    @Test
+    @Transactional
```

Comment:
what's the purpose of Transactional annotation in mock test?

Reply:
I will fix it in the next commit